opentracing/standard/raw.py

```diff
@@ -30,13 +30,13 @@ def __init__(self, trace_context, operation_name):
         self.operation_name = operation_name
         self.start_time = None
         self.duration = None
-        self.tags = None  # array of (key,vale) pairs when initialized
+        self.tags = None  # dict when initialized
         self.logs = None  # array of RawLog's when initialized
 
-    def add_tag(self, key, value):
+    def set_tag(self, key, value):
         if self.tags is None:
-            self.tags = []
-        self.tags.append((key, value))
```

Comment:
I think @yurishkuro wanted to take advantage of the undefined-ness of setting multiple values for a single key (e.g., for "parent_id").